tardis/model/base.py

```diff
@@ -73,6 +73,15 @@ def __init__(self, velocity, homologous_density, abundance, time_explosion,
         self.homologous_density = homologous_density
         self._abundance = abundance
         self.time_explosion = time_explosion
+
+        self.raw_abundance = self._abundance
+        if isotope_abundance is not None:
+            self.raw_isotope_abundance = isotope_abundance
+            self._abundance = self.raw_isotope_abundance.as_atomic_numbers(
```

Comment:
As your profiling showed, executing this for 100 elements takes less than 0.02 seconds. In my opinion it's okay to call it every time. especially as it should always reflect the current state of the model.

Currently tardis objects are not designed to be changed, nevertheless if one were to change `time_explosion` on the model, the isotopic abundances have to be recalculated. The easiest way to achieve this is by doing this (very fast) computation every time. I think you can count the number of accesses to `model.abundance` during a typical tardis run with one hand.